athletics.lic

```diff
@@ -121,19 +126,21 @@ class Athletics
         end
         return if remaining_attempts.zero?
       end
-      Flags.add('ct-climbing-finished', 'You finish practicing your climbing')
+      Flags.add('ct-climbing-finished', 'You finish practicing your climbing', 'The rope\'s will quickly fades away')
       Flags.add('ct-climbing-combat', 'You are engaged in combat')
-      bput("climb practice #{target}", 'You begin to practice ')
+      fix_standing
+      bput("play #{UserVars.song} on zills", 'You begin', 'You\'re already playing a song') if target.eql?('climbing rope') 
```

Comment:
I'm copying it and cutting out crossing-trainer specific lines. It seems like this logic can be commonized, but I'll save that for a future commit or PR.